docker/edge-alpine.Dockerfile

```diff
@@ -4,6 +4,7 @@ WORKDIR /app
 ADD .yarn ./.yarn
 ADD yarn.lock package.json .yarnrc.yml ./
 RUN yarn workspaces focus --all --production
+RUN if [ "$(uname -m)" = "armv7l" ]; then npm install bcrypt better-sqlite3 --build-from-source; fi
```

Comment:
Where is `--build-from-source` defined?

Reply:
Sorry, I'm not quite understand the question.

Coming GitHub log, I tried explicitly run `npm install` but no avail (even GitHub log is silent), for some reason it won't rebuild automatically. Thus I'm just searching how to rebuild npm modules from source, and I got https://github.com/mapbox/node-pre-gyp/issues/58

From ~4 minutes build time to ~16 minutes, definitely something is happened.

Reply:
Ok, I see — that option is interpreted by `node-pre-gyp`, not `npm install`.